src/systems/physics/Physics.cc

```diff
@@ -161,38 +161,39 @@ class gz::sim::systems::PhysicsPrivate
   /// implement to be supported by this system.
   /// New features can't be added to this list in minor / patch releases, in
   /// order to maintain backwards compatibility with downstream physics plugins.
-  public: struct MinimumFeatureList : gz::physics::FeatureList<
-          gz::physics::FindFreeGroupFeature,
-          gz::physics::SetFreeGroupWorldPose,
-          gz::physics::FreeGroupFrameSemantics,
-          gz::physics::LinkFrameSemantics,
-          gz::physics::ForwardStep,
-          gz::physics::RemoveModelFromWorld,
-          gz::physics::sdf::ConstructSdfModel,
-          gz::physics::sdf::ConstructSdfWorld,
-          gz::physics::GetLinkFromModel,
-          gz::physics::GetShapeFromLink
+  public: struct MinimumFeatureList : physics::FeatureList<
+          physics::FindFreeGroupFeature,
+          physics::SetFreeGroupWorldPose,
+          physics::FreeGroupFrameSemantics,
+          physics::LinkFrameSemantics,
+          physics::ForwardStep,
+          physics::RemoveModelFromWorld,
+          physics::sdf::ConstructSdfLink,
```

Comment:
revert this `ConstructSdfLink` line

Reply:
[ffe2da0](https://github.com/gazebosim/gz-sim/pull/1794/commits/ffe2da087fb41925a0fab13ee063774c1c93cae0)